src/app/loadout-drawer/loadout-apply.ts

```diff
@@ -816,29 +815,37 @@ function applySocketOverrides(
             }
             if (modHash) {
               const mod = defs.InventoryItem.get(modHash) as PluggableInventoryItemDefinition;
-              modsForItem.push({ socketIndex, mod });
+              // We explicitly set sockets that aren't in socketOverrides to the default plug for subclasses
```

Comment:
```suggestion
              // We explicitly set sockets that aren't in socketOverrides to the default plug for subclasses abilities
```

Reply:
FYI I am going to try and get something up to change this behaviour in the next couple of days, like we discussed.